satpy/composites/__init__.py

```diff
@@ -589,6 +589,17 @@ def __call__(self, projectables, optional_datasets=None, **info):
 class NIRReflectance(CompositeBase):
     """Get the reflective part of NIR bands."""
 
+    def __init__(self, **kwargs):
```

Comment:
You can make max_sza explicit:
```suggestion
    def __init__(self, max_sza=None, **kwargs):
```